sdk/storage/Azure.Storage.Common/src/Shared/Constants.cs

```diff
@@ -73,6 +73,11 @@ internal static class Constants
         /// </summary>
         public const bool DefaultTrimBlobNameSlashes = true;
 
+        /// <summary>
+        /// Backwards compatible default value for whether to include the StorageRequestValidationPipelinePolicy.
+        /// </summary>
+        public const bool DontIncludeStorageRequestValidationPipelinePolicy = true;
```

Comment:
As per my comment above, the default value will change to false and you will probably want to rename this to something like "DefaultDisableClientRequestIdValidation"